Description: Force LC_ALL=C environment variable when grepping the output of lpstat.
 Permits testing in non-"C" environments.
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Last-Update: 2016-07-03
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -846,7 +846,7 @@
 	fi
 done
 
-description="`../systemv/lpstat -l -p Test1 | grep Description | sed -e '1,$s/^[^:]*: //g'`"
+description="`LC_ALL=C ../systemv/lpstat -l -p Test1 | grep Description | sed -e '1,$s/^[^:]*: //g'`"
 if test "x$description" != "xTest Printer 1"; then
 	echo "Failed, printer-info for Test1 is '$description', expected 'Test Printer 1'." >>$strfile
 	echo "FAIL (got '$description', expected 'Test Printer 1')"
